src/QuadForm/Quad/GenusRep.jl

```diff
@@ -511,7 +511,7 @@ function maximal_norm_splitting(L, p)
     end
   end
 
-  @assert all(let G = G; k -> nrows(G[k]) in [1,2]; end, 1:length(G))
+  @assert all(k -> nrows(G[k]) in [1,2], 1:length(G))
```

Comment:
Did you check with `code_warntype`? This is added to work around a julia bug.

Reply:
I ran the `code_warntype` and notice wrong appears.. The assertion works also (like if I change some values it returns an error, and does nothing otherwise). So it seems to be fixed (?)

Reply:
ok